app/helpers/info_request_helper.rb

```diff
@@ -236,4 +236,48 @@ def status_text_awaiting_description_other(info_request, new_responses_count)
        user: user_link_for_request(info_request))
   end
 
+  def attachment_link(incoming_message, attachment)
+    img_filename = "icon_#{attachment.content_type.sub('/', '_')}_large.png"
+    full_filename = File.expand_path(Rails.root.join('app',
+                                                     'assets',
+                                                     'images',
+                                                     img_filename))
+    image = if File.exist?(full_filename)
+      img_filename
+    else
+      "icon_unknown.png"
+    end
+
+    link_to image_tag(image, :class => "attachment__image",
+                             :alt => "Attachment"),
+                      attachment_path(incoming_message, attachment)
```

Comment:
I'm not sure if this is actually the line Hound is complaining about above, but I think `attachment_path` should be aligned to `image_tag`? (It's a param to `link_to` right?)